src/openzaak/components/besluiten/models.py

```diff
@@ -157,6 +157,29 @@ class Besluit(AuditTrailMixin, APIMixin, models.Model):
         help_text="De datum tot wanneer verweer tegen het besluit mogelijk is.",
     )
 
+    _previous_zaak_url = models.URLField(
+        _("externe previous zaak"), blank=True, max_length=1000,
+    )
+    _previous_zaak = models.ForeignKey(
+        "zaken.Zaak",
+        on_delete=models.PROTECT,
+        null=True,
+        blank=True,  # een besluit kan niet bij een zaak horen
+        related_name="previous_zaak",
+    )
+    previous_zaak = FkOrURLField(
```

Comment:
not sure why you're adding these as actual model fields?

Reply:
`previous_zaak` fields are removed from db